### PR TITLE
ANC - Progress Report: CHW Activity pane

### PR DESCRIPTION
**Pull request by anvmn**
#381

Replaced https://github.com/Gizra/ihangane/issues/2896 from private repo

### CODE REVIEW

**Comment by anvmn** [comment]
Selection_957

**Comment by anvmn** [comment]
@nedSaf please review

File: client/src/assets/scss/_variables.scss [hunk 1]
```diff
@@ -24,7 +24,7 @@ $color-lighter-blue:        #E0E7FF;
 $color-white:               #FFFFFF;
 $color-svg-grid:            #8097B1;
 $color-orange-dark:         #F7931E;
-$color-blue:                #4C06FF;
```
Comment (nedSaf) [inline client/src/assets/scss/_variables.scss:27]: Just want to make sure, are those above the only uses of `$color-blue`?

**Review by nedSaf — APPROVED** [review]
looks good, just one question for the peace of mind :)

File: client/src/assets/scss/_variables.scss (hunk 1 above)
Comment (anvmn) [reply client/src/assets/scss/_variables.scss:27]: @nedSaf 

Yes:

Selection_986

**Pull request merged**